db/db.js

```diff
@@ -35,3 +35,15 @@ exports.checkCategoryExists = (category) => {
         }
     });
 }
+
+exports.checkCommentExists = (comment_id) => {
+    return pool.query(
+        ` SELECT * FROM comments
+        WHERE comments.comment_id = $1;`, [comment_id]
+    )
+    .then((res) => {
+        if (res.rows.length === 0) {
+            return Promise.reject({status: 404, msg: `Comment does not exist`})
+        }
+    });
+}
```

Comment:
In this case there is no need to make a second query to check if the comment exists.